.github/workflows/packaging.yml

```diff
@@ -77,10 +77,17 @@ jobs:
             -u ${RWS_AUTH} \
             -F product=${PRODUCT_NAME}"
 
+        # We don't want to try to print secrets to the log, but we want
+        # to print a "curl" command to see what's going on.
+          CURL_CMD_ECHO="curl -LfsS \
+            -X PUT ${RWS_URL_PART}/${OS}/${DIST} \
+            -F product=${PRODUCT_NAME}"
```

Comment:
I would add `-u ***` or `-u <...>` or `-u ${RWS_AUTH}` explicitly. The latter also allows to define the variable and copy-paste the command.

Reply:
I choose `-u ***`